generator/README.md

```diff
@@ -127,6 +127,17 @@ modules:
                              #   EnumAsStateSet: An enum with a time series per state. Good for variable low-cardinality enums.
 ```
 
+### EnumAsInfo and EnumAsStateSet
+
+Sometimes it'll be necessary to specify an override for a metric because they are
```

Comment:
I would prefer this sentence:

```
SNMP contains the concept of integer indexed enumerations (Enums). There are two ways to represent these strings in Prometheus. They can be "info" metrics, or they can be "state sets". SNMP does not specify which should be used, and it's up to the use case of the data. Some users may also prefer the raw integer value, rather than the string.

In order to set enum integer to string mapping, you must use one of the two overrides.
```